src/component/calc.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calc extends React.Component {
```

Comment:
- [ ] Like the previous reviewer mentioned, kindly use functional components here as it is a requirement and is considered best practice
- [ ] Kindly, do not disable linters as they help us write better code.

Reply:
Thank you for the comment!